src/Operation/Map.php

```diff
@@ -28,7 +28,7 @@ final class Map extends AbstractOperation
      *
      * @template V
      *
-     * @return Closure(callable(T=, TKey=, Iterator<TKey, T>=): V): Closure(Iterator<TKey, T>): Generator<TKey, V>
+     * @return Closure(callable(T=, TKey=, Iterator<TKey, T>=): mixed): Closure(Iterator<TKey, T>): Generator<TKey, mixed>
```

Comment:
if we're changing this here then it should also be reflected in:
- Mapable
- AsyncMap
- AsyncMapable

In general any type changes on the operations should match the interfaces; we don't want to have discrepancies between them

Reply:
same for the scans and others in this PR

Reply:
I think it's ready now :)